File: _src/05/Chapter5.CSharp/Models/Company.cs

using System.Collections.Generic;

namespace Chapter5.CSharp.Models
{
public class Company
{
    public Company()
    {
        Persons = new HashSet<Person>();
        Address = new Address();
    }
    public int CompanyId { get; set; }
    public string CompanyName { get; set; }
    public Address Address { get; set; }
    public ICollection<Person> Persons { get; set; }
}
}

File: _src/05/Chapter5.VB/Models/Company.vb
Namespace Models
    Public Class Company
        Public Sub New()
            Persons = New HashSet(Of Person)
            Address = New Address()
        End Sub

        Property CompanyId() As Integer
        Property CompanyName() As String
        Property Address() As Address
        Overridable Property Persons() As ICollection(Of Person)
    End Class
End Namespace